webtunnelclient/client.go

```diff
@@ -153,10 +155,31 @@ func (w *WebtunnelClient) SetServer(serverIPPort string, secure bool, wsDialer *
 	w.wsDialer = wsDialer
 }
 
+// getUsernameHostname gets the username and hostname of the client
+func (w *WebtunnelClient) getUserInfo() (string, error) {
+
+	username, err := user.Current()
+	if err != nil {
+		return "", err
+	}
+
+	hostname, err := os.Hostname()
+	if err != nil {
+		return "", err
+	}
+	return username + " " + hostname, nil
```

Comment:
username is not type string. its *user.User. You probably want username.Username which is string

Reply:
Done.